extensions/wikia/FlowTracking/scripts/initCreatePageTracking.js

```diff
@@ -77,9 +82,9 @@ require([
 			}
 		});
 
-		mw.hook('ve.afterVEInit').add(function(veEditUri) {
-			if (!!mw.config.get( 'wgArticleId' )) {
-				veEditUri.extend( { flow: createButtonFlow } );
+		mw.hook('ve.afterVEInit').add(function (veEditUri) {
+			if (!mw.config.get('wgArticleId')) {
```

Comment:
If we use `!!` this condition is true when article exists and we want to add param only if article doesn't exist